src/templates/layouts/event/index.tsx

```diff
@@ -47,6 +72,18 @@ export const Event = ({
     .filter(Boolean)
     .join(', ')
 
+  const createMailToLink = (emailCTA, title, mostRecentDate, linkPath) => {
```

Comment:
i think i would move this to `utils/event.ts` also, it feels a little weird to me that the CTA is formatted in a util file but this isn't